prowler/providers/gcp/services/compute/compute_service.py

```diff
@@ -21,11 +26,29 @@ def __init__(self, audit_info):
         self.__get_regions__()
         self.__get_projects__()
         self.__get_zones__()
-        self.__get_instances__()
+        self.__zone_threading_call__(self.__get_instances__)
         self.__get_networks__()
-        self.__get_subnetworks__()
+        self.__region_threading_call__(self.__get_subnetworks__)
         self.__get_firewalls__()
 
+    def __zone_threading_call__(self, call):
+        threads = []
+        for zone in self.zones:
+            threads.append(threading.Thread(target=call, args=(zone,)))
+        for t in threads:
+            t.start()
+        for t in threads:
+            t.join()
+
+    def __region_threading_call__(self, call):
+        threads = []
+        for region in self.regions:
+            threads.append(threading.Thread(target=call, args=(region,)))
+        for t in threads:
+            t.start()
+        for t in threads:
+            t.join()
+
```

Comment:
Please, move this to the new `GCPService` class.

Reply:
Done